castle/test/client_test.py

```diff
@@ -103,58 +110,46 @@ def test_track_tracked_true(self):
             json=response_text,
             status=200
         )
-        client = Client(request(), {})
+        client = Client.from_request(request(), {})
         options = {'event': '$login.authenticate', 'user_id': '1234'}
         self.assertEqual(client.track(options), response_text)
 
     def test_track_tracked_false(self):
-        client = Client(request(), {})
+        client = Client.from_request(request(), {})
         client.disable_tracking()
         self.assertEqual(client.track({}), None)
 
     def test_disable_tracking(self):
-        client = Client(request(), {})
+        client = Client.from_request(request(), {})
         client.disable_tracking()
         self.assertEqual(client.do_not_track, True)
 
     def test_enable_tracking(self):
-        client = Client(request(), {})
+        client = Client.from_request(request(), {})
         client.disable_tracking()
         self.assertEqual(client.do_not_track, True)
         client.enable_tracking()
         self.assertEqual(client.do_not_track, False)
 
-    def test_tracked_true(self):
-        client = Client(request(), {})
+    def test_dont_tracked_false(self):
```

Comment:
i think you want `test_do_not_track_false`

Reply:
ahh that naming .. will review that and correct 👍